src/ds5/advanced_mode/advanced_mode.cpp

```diff
@@ -80,7 +80,7 @@ namespace librealsense
                 default_420(p);
                 break;
             default:
-                throw invalid_value_exception(to_string() << "apply_preset(...) failed! Invalid product ID (" << device_pid << ")");
+                throw invalid_value_exception(to_string() << "apply_preset(...) failed! Given device doesn't support Default Preset (pid=" << device_pid << ")");
```

Comment:
Please wrap device_pid with  std::hex/std::dec to use the regular representation